lib/components/godoc-panel-view.js

```diff
@@ -0,0 +1,39 @@
+/** @babel */
+/** @jsx etch.dom */
+
+import etch from 'etch'
+
+export default class GodocPanelView {
```

Comment:
btw: I found the problem why after some time the panel does not work anymore (as you mentioned in the commit message 4227b85). Because etch destroys the DOM node. This is actively prevented by `EtchComponent.destroy` but not in your `GodocPanelView`, so inheriting from `EtchComponent` would solve this problem too 😉